### DISCUSSION TITLE
LibriSpeech Transducer CheckPoint Missing

### DISCUSSION THREAD

**Discussion by serchsm** [speechbrain/speechbrain#1771] [General]
Hello,

 I was wondering why the checkpoint for the LibriSpeech Transducer is missing, see [recipe](https://github.com/speechbrain/speechbrain/tree/develop/recipes/LibriSpeech/ASR/transducer#librispeech-100h-results) and [drive](https://drive.google.com/drive/folders/17kEW0crU3tyP-8-u5TeoFom4ton_B-j2).  
 I am trying to do transfer learning and I was hoping to use this checkpoint. Any chance it could be posted?

Thank you!

**Comment by TParcollet** [comment]
Hi, we had some issues in the past... Let me do something: I will try to train the whole model again on libri (the one reported is only libri 100), and upload the checkpoint. It might takes more than a week though

**Reply by serchsm** [reply]
Ok. Thank you!
